include/net/zzhc.rst

```diff
@@ -0,0 +1,74 @@
+.. _lib_zzhc:
```

Comment:
This library is included on the list of "Libraries for FOTA updates" on the Libraries section main page.
How does this library relate to the FOTA update libraries listed there? Could you clarify this in the documentation, @pkchan ?

Reply:
`zzhc.rst` has nothing to do with FOTA. The file is in the `include\net` directory, which means it's the document for network libraries. I do not know why `include\net` has the name "**Libraries for FOTA updates**". It should be changed to "**Libraries for networking**" instead.

Reply:
Per request, in `doc\nrf`, added section "**Libraries for networking**", which includes `zzhc`.